sdk/test-utils/recorder-new/README.md

```diff
@@ -14,9 +14,9 @@ Feature work is being tracked at [#15829](https://github.com/Azure/azure-sdk-for
 
 - Run this command
 
-  > `docker run -v /workspaces/azure-sdk-for-js/:/etc/testproxy -p 5001:5001 -p 5000:5000 azsdkengsys.azurecr.io/engsys/testproxy-lin:latest`
+  > `docker run -v /workspaces/azure-sdk-for-js/:/srv/testproxy -p 5001:5001 -p 5000:5000 azsdkengsys.azurecr.io/engsys/testproxy-lin:latest`
```

Comment:
@HarshaNalluru these are the only places that I could find with `etc/testproxy` noted. Can you throw me a sign-off if you agree with these changes?